### PR TITLE
Style updates part 1

### LINKED ISSUE #854: Visual Style Updates
* Edit 1.5 design tokens
- main areas
    - heading and body text sizes and bold levels
    - typefaces
    - colors - check in with @smombartz  to ensure they're finalized before making changes.
    - Button variants
        - primary, secondary, form, destructive, ... (organize variants by functionality)
        - filled or outlined
        - small, medium, large
    - add these to stitches first, then make Material-UI theme file depend on stitches
- Update components to refer to variants/tokens in theme instead of specific values (un-adHoc-ing)
- Create new components from [Design Library](https://www.figma.com/file/ZHiNW0Wf2bj2VHVIqpjMuh/Design-Library?node-id=105%3A2519)  (if they dont already exist)
- change all the button colors — should be covered by the changes above
    - all buttons on vaults page → blue
    - all submit buttons on all form modals → blue
    - next step button on allocation flow → blue
    - all 4 buttons on circle admin page → blue
    - vouching page → blue
    - grey buttons on circle admin settings modal → grey outline, no icons

### PR DESCRIPTION
**Pull request by levity**
Part 1 of the work for #854.

This cleans up a ton of the color configuration while making only small changes to the end-user UX, so that it can be merged ASAP.

User-facing changes
- Some color values change (e.g. red is more saturated, light blue is more teal, header is darker)
- The links on circle rows in the overview page are not buttons anymore (this reflects the 1.5 figma)
- Some text changed size slightly

Non-user-facing changes
- the landing page's theme is decoupled from the app's theme
- in the app, src/ui/colors is the single source of color truth
- and stitches.config implements a short list of semantic color names & font size tokens
- components use only semantic tokens
  - exception: the map's colors were hardcoded and were not changed

### CODE REVIEW

**Comment by levity** [comment]
@Zemm111 @reeserj when you have time, can you browse through https://coordinape-git-854-style-updates-coordinape.vercel.app/ and let me know if anything looks weird? this contains lots of little adjustments to colors, but there shouldn't be anything a normal user would find disruptive

**Comment by reeserj** [comment]
@levity 
1. Saving my allocations took a long time
2. The history bug seems to be present here too
3. I think the hover color here 
image
is different than the hover color here
image


The colors to look good to me!

**Comment by levity** [comment]
> @levity
> 
> 1. Saving my allocations took a long time
> 2. The history bug seems to be present here too
> 3. I think the hover color here
>    image
>    is different than the hover color here
>    image
> 
> The colors to look good to me!

ahh, good catch on #3, thanks!

**Review by teeolendo — APPROVED** [review]
LGTM

File: src/stitches.config.ts [hunk 1]
```diff
@@ -50,6 +50,34 @@ const spaces = {
...
+
+  background: figmaColors.grey1,
+  surface: figmaColors.grey2,
+  border: figmaColors.grey4,
+  focusedBorder: figmaColors.grey8,
+
+  link: figmaColors.blue12,
+  text: figmaColors.grey20,
+  headingText: figmaColors.grey16,
+  secondaryText: figmaColors.grey12,
```
[the remark this answers is not in the document]
Comment (teeolendo) [reply src/stitches.config.ts:78]: Just realized that alert = red

**Pull request merged**